content/etc/icingaweb2/config.ini

```diff
@@ -0,0 +1,10 @@
+[global]
+show_stacktraces = "1"
+config_backend = "db"
+config_resource = "icingaweb_db"
+module_path = "/usr/share/icingaweb2/modules:/etc/icingaweb2/modules"
```

Comment:
Sounds good!  Right now, the only example of a docker from the docker-container would be https://github.com/findmypast/icingaweb2-module-graphite right?